test/unit/ui/access_store_ui_app.py

```diff
@@ -0,0 +1,22 @@
+"""
+UI app for access store for testing only
+"""
+
+from IPython import get_ipython
+
+from exasol.nb_connector.ui import access_store_ui
+
+sb_store_file = access_store_ui.DEFAULT_FILE_NAME
+
+ui = access_store_ui.get_access_store_ui()
+
+password = ui.children[0].children[2].children[1]
+password.value = "password"
+
+open_button = ui.children[1]
+open_button.click()
+
+ipython = get_ipython()
+ipython.run_line_magic("store", "-r")
```

Comment:
I was able to talk to @tomuben who was so kind to forward the following comment from @tkilias : 

> the store magic probably does not work in our CI
> The advice to create the iPython file (if it doesn't exist) also did not help 
> 
> We now recommend replacing the store magic by open/write/read 


Comparing the notebook [first_steps.ipynb](https://github.com/exasol/ai-lab/blob/main/exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/first_steps.ipynb#L669):
```
    "%%writefile first_steps/text_file.txt\n",
    "Hello World!"
```

we maybe can use `ipython.run_line_magic("readfile", "filename.txt")`

See also https://ipython.readthedocs.io/en/stable/interactive/magics.html